src/components/Header.js

```diff
@@ -3,16 +3,33 @@ import { Outlet, Link } from 'react-router-dom';
 
 const Header = () => (
   <>
-    <nav className="flex flex-row py-4 justify-between px-4">
-      <div className="text-3xl">Bookstore CMS</div>
```

Comment:
Well done with adding styling to your CMS Bookstore. Your design abilities are truly commendable! 🔥💯 

Remember by "styling should **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/)", that includes fonts too. Kindly use the correct `font-family` for the header elements and book card elements.

1. According to the Zepplin design, the logo `Bookstore CMS` and nav links `Books` and `Categories` should have the font family `Montserrat`.

![image](https://github.com/abel-tefera/bookstore/assets/38088818/875b5780-88ac-4718-80e4-0df20bff551e)

You can import `Montserrat` from Google Fonts.

![image](https://github.com/abel-tefera/bookstore/assets/38088818/11bfb1f8-bdd6-4319-baaa-fc891413b7c6)

2. According to the Zepplin design, all the `Book cards` should have the font family `RobotoSlab`. 

![image](https://github.com/abel-tefera/bookstore/assets/38088818/1e28cbf2-70a8-409f-83d3-4c9e837c8d40)

You can import the `Roboto Slab` font from Google Fonts.

![image](https://github.com/abel-tefera/bookstore/assets/38088818/2d5588aa-3bed-44b0-816c-5745ef94fd22)